hail/src/test/scala/is/hail/expr/ir/EmitStreamSuite.scala

```diff
@@ -193,6 +193,19 @@ class EmitStreamSuite extends HailSuite {
     assert(range(null) == null)
   }
 
+  @Test def testEmitSeqSample(): Unit = {
+    val seq = evalStream(
+      SeqSample(
+        I32(20),
+        I32(5),
+        false
+      )
+    ).map(_.asInstanceOf[Int])
+    assert(seq.size == 5)
+    assert(seq.toSet.size == 5)
+    assert(seq.forall(e => e >= 0 && e < 20))
```

Comment:
It's always hard to test random methods, but we should probably have something to check that this has the right statistical properties. I would suggest repeating `SeqSample(N, n)` a large number of times (maybe compile once and run repeatedly), where n is 2 or 3, and N is smallish, then count how many times each possible two element subset gets sampled in an NxN matrix. If the sampler is uniform, they should all be nearly equal.